src/ucs/config/global_opts.c

```diff
@@ -47,6 +47,7 @@ ucs_global_opts_t ucs_global_opts = {
     .stats_filter          = { NULL, 0 },
     .stats_format          = UCS_STATS_FULL,
     .rcache_check_pfn      = 0,
+    .rcache_pfn_max_pages  = 1,
```

Comment:
can we unite these to single var, so by default number of pages is 0 which means to not check at all

Reply:
do you mean change variable type? but this change will break backward compatibility.

Reply:
yes, since this is a debug variable we don't really have to maintain compatibility